src/main/scripts/org/reaktivity/specification/nukleus/oauth/control/resolve/with.roles/controller.rpt

```diff
@@ -18,7 +18,7 @@ property nukleiR05C ${nuklei:directory("target/nukleus-itests").controlCapacity(
 
 property controlR05C ${nukleiR05C.control()}
 
-property authorization 0x0001_000000000000cL
+property authorization 0x0001_000000000007L
```

Comment:
Let's inline `authorization` directly with the value below, to eliminate temptation to parameterize this test with a different `authorization` property value when testing against implementation.